SRC/PORT.CPP

```diff
@@ -78,6 +78,11 @@ static void get_render_rect( SDL_Rect *rect )
 
 static int create_surface()
 {
+    if ( surface )
+    {
+        SDL_FreeSurface(surface);
```

Comment:
> 
> 
> This oughta set `surface = NULL` too, to be absolutely clean about it :)

I thought about it but because next line assigns to it again, hopefully a valid pointer or NULL in case of a failure, I decided to save a line. ;)